style.css

```diff
@@ -0,0 +1,179 @@
+/**
+// General Styles
+**/
+* {
+    margin: 0;
+    padding: 0;
+  /*   box-sizing: border-box; */
+  /*   background-color: #151515;  Gives everything a black background including text */
+  }
+  
+  body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: #d3d3d3;
+    background-color: #151515;
+  }
+  
+  h2 {
+    font-size: 1em;
+  }
+  
+  
+  button {
+    color: inherit;
+  }
+  
+  .main-content {
+    margin-top: 50px;
+  }
+  
+  /**
+  // Header Styles
+  **/
+  
+  
+  header h1 {
+    font-size: 1em;
+    color: #d3d3d3;
+  }
+  
+  /**
+  // Navbar Styles
+  **/
+  
+  .navbar {
+    background-color: #090909;
+    color: #d3d3d3;
+    position: absolute;
+    width: 100%;
+    padding: 10px;
+  }
+  
+  .navbar_dropdown_content {
+    position: absolute;
+    display: none;
+    background-color: black;
+    // color: #d3d3d3;
+    text-decoration: none;
+  }
+  
+  .navbarlist {
+    list-style-type: none;
+    display: inline;
+    background-color: #090909;
+    color: #d3d3d3;
+  }
+  
+  /** #trending, #comingsoon, #action, #comedy, #thriller, #horror, #romance {
+  
+  }
+  **/
+  
+  /* *
+  // Utility Styles?
+  *
+   */
+  .row {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 10px;
+  }
+  
+  .img-slider {
+    display: flex;
+    margin-bottom: 25px;
+  }
+  
+  .images {
+    display: flex;
+    justify-content: center;
+    min-width: 320px;
+  }
+  
+  .event-image {
+    width: 100%;
+    max-height: 450px;
+  }
+  
+  .events{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background-size: cover;
+    background-position: center;
+    color: white;
+    font-size: 1em;
+    margin-bottom: 10px;
+    width: 85%;
+  }
+  
+  .event-button {
+    font-size: 19px;
+    width: 200px;
+    border-radius: 20px;
+    color: white;
+    background-color: purple;
+  }
+  
+  .bottom {
+    display: flex;
+    justify-content: center;
+    justify-content: space-between;
+  }
+  
+  footer-column {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+  
+  .no-dot {
+    list-style-type: none;
+  }
+  
+  a:link {
+    text-decoration: none;
+    color: white;
+  }
+  
+  a:hover {
+    color: purple;
+  }
+  
+  .icon {
+    width: 25px;
+  }
+  
+  .main-wrapper {
+      width: 90%;
+      margin: 0 auto;
+  }
+  
+  .move-over {
+    margin-left: 40px;
+  }
+  
+  // .hidden {
+  //     display: none;
+  // }
+  
+  /* media queries */
+  @media screen and (max-width: 340px){
```

Comment:
I want to say I did this because at different points things would start to look ugly so I broke it up plus the font-size would either shrink the header to much or the paragraphs not enough